android/modules/ui/src/java/ti/modules/titanium/ui/UIModule.java

```diff
@@ -178,6 +178,9 @@
 	@Kroll.constant public static final int INPUT_TYPE_CLASS_NUMBER = InputType.TYPE_CLASS_NUMBER;
 	@Kroll.constant public static final int INPUT_TYPE_CLASS_TEXT = InputType.TYPE_CLASS_TEXT;
 
+	@Kroll.constant public static final int VIEW_GONE = View.GONE;
+	@Kroll.constant public static final int VIEW_INVISIBLE = View.INVISIBLE;
```

Comment:
Thinking about it, we might want to rename them to `Ti.UI.HIDDEN_BEHAVIOR_GONE` and `Ti.UI.HIDDEN_BEHAVIOR_INVISIBLE`. What do you think? And please add + validate the docs for the constants, they are not documented, yet.